src/API/Site/Controllers/Ads/AccountController.php

```diff
@@ -86,6 +86,33 @@ public function register_routes(): void {
 				],
 			]
 		);
+
+		$this->register_route(
+			'ads/accepted-customer-data-terms',
+			[
+				[
+					'methods'             => TransportMethods::READABLE,
+					'callback'            => $this->get_accepted_customer_data_terms_callback(),
+					'permission_callback' => $this->get_permission_callback(),
+				],
+			]
+		);
+
+		$this->register_route(
+			'ads/enhanced-conversion-status',
+			[
+				[
+					'methods'             => TransportMethods::EDITABLE,
+					'callback'            => $this->update_enhanced_ads_conversion_callback(),
+					'permission_callback' => $this->get_permission_callback(),
```

Comment:
As mentioned in a previous comment, the 'args' are missing here to enforce the schema of what parameters we'd accept in this endpoint.